packages/kit/src/template.ts

```diff
@@ -113,6 +113,13 @@ export async function writeTypes (nuxt: Nuxt) {
 
   const rootDirWithSlash = withTrailingSlash(nuxt.options.rootDir)
 
+  const modules = nuxt.options._installedModules
+    .filter(m => m.entryPath)
+    .map(m => m.entryPath.startsWith(rootDirWithSlash)
+      ? m.entryPath.split('/index.ts')[0]
+      : rootDirWithSlash + 'node_modules/' + m.entryPath
```

Comment:
we might need to resolve the module path with `tryResolveModule` or similar instead - in a monorepo or in non-hoisted pnpm setup we can't necessarily assume that `~/node_modules/<module-name>` is correct.